core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -1,11 +1,11 @@
 <figure class="account-image js-file-upload">
     {{#unless hasUploadedImage}}
-        <div class="placeholder-img" style={{defaultImage}}></div>
-
         {{#if hasEmail}}
```

Comment:
`hasEmail` will be true as soon as anything is typed in the email field - removing the placeholder image as soon as that happens definitely isn't the desired behaviour